src/App.tsx

```diff
@@ -15,36 +23,99 @@ export const goodsFromServer = [
   'Garlic',
 ];
 
-export const App: React.FC = () => {
+enum SortBy {
+  'default',
+  'alpabet',
+  'length',
+}
+
+function getGoods(goodsFromServList: string[]): GoodsList {
```

Comment:
The function name `getGoods` does not start with a verb which makes it less clear what the function does. Consider renaming it to something like `fetchGoods` or `retrieveGoods` to better reflect its purpose.